### PR TITLE
[DOCS] Updated iOS native UI component events documentation

### PR DESCRIPTION
**Pull request by jeanregisser**
Documentation on iOS about triggering events from native components was outdated.

Let me know what you think.

### CODE REVIEW

**Comment by jeanregisser** [comment]
I also wanted to add a note about the difference between `RCTBubblingEventBlock` and `RCTDirectEventBlock` but wasn't too sure which one to recommend depending on the use case.

Could someone shed some light on this?

**Comment by ide** [comment]
cc @nicklockwood

**Comment by facebook-github-bot-4** [comment]
@knowbody would you mind taking a look at this pull request? It's been a while since the last commit was reviewed.

**Comment by knowbody** [comment]
lgtm!

**Comment by knowbody** [comment]
@facebook-github-bot shipit

**Comment by jeanregisser** [comment]
Did the shipit command work?

**Comment by knowbody** [comment]
I think not, let me try again

**Comment by knowbody** [comment]
@facebook-github-bot shipit

**Comment by knowbody** [comment]
unless I don't have permission, /cc: @mkonicek

**Comment by mkonicek** [comment]
Yup you don't have the permission.

@facebook-github-bot shipit

**Comment by facebook-github-bot-7** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/133346907061611/int_phab) to review.
